tests/lib/rules-preprocessor/gjs-gts-processor-test.js

```diff
@@ -326,6 +329,83 @@ describe('lint errors on the exact line as the <template> tag', () => {
 });
 
 describe('multiple tokens in same file', () => {
+  // TODO: this test still fails, needs to rework the token searching logic
```

Comment:
I see passing tests?

Reply:
forgot to remove this comment now that I fixed it 🤦